From ddd4ecdc3c0485f5103e84a54bdaa406e3e645ad Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Date: Fri, 25 Apr 2014 22:06:23 +0200
Subject: unit_tests: increment test-timeout from 40secs to 5mins

I see now and then that tests on the slower archs (arm*, mips) fail due to
the timeout restriction. Often a second build even on the same buildd
succeeds. I think that the buildd is either close to the limit or has
something else (another sbuild running) going so it does not finish the
test in 40secs.
Therefore I raise the bar to a 5 minutes limit. The worst thing that can
happen is, that it takes more than 4 minutes before an enless loop will
be detected.

Signed-off-by: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
---
 unit_tests/preload_run.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/unit_tests/preload_run.sh b/unit_tests/preload_run.sh
index 8b8c8e3..365b3a8 100755
--- a/unit_tests/preload_run.sh
+++ b/unit_tests/preload_run.sh
@@ -5,5 +5,5 @@
 ulimit -c 0 || true
 LD_PRELOAD=$LIBPRELOAD
 export LD_PRELOAD
-export CK_DEFAULT_TIMEOUT=40
+export CK_DEFAULT_TIMEOUT=300
 exec $@
